### DISCUSSION TITLE
DoH over non standard port

### DISCUSSION THREAD

**Discussion by datatype** [paulmillr/encrypted-dns#58] [General]
Unfortunately I need to create a profile for an Inhouse (non-public) DoH proxy. Creating the profile was quite easy by modifying the google profile provided here, but I stuck at the point that I need to run the DoH proxyy on another port than 443. 
Can I define the port somewhere in profile?

**Comment by paulmillr** [comment]
did you try something like this?

```
				<key>ServerURL</key>
				<string>https://security.cloudflare-dns.com:42091/dns-query</string>
```

**Comment by datatype** [comment]
Yep, it's ignored :-(  The iPad still tries to reach the DoH under :443...

**Comment by ghost** [comment]
@paulmillr Since it's not a bug in .mobileconfig file I ask this issue be closed.
We cannot fix it only apple can so I vote to close it but it's up to you.

**Comment by cyb3rz3us** [comment]
I have observed the same issue with the port being ignored by the iOS device but fortunately, there's a nice little (and free) app called **DNSCloak** that provides relief here.  Easy setup and works like a charm.
